packages/core-components/src/components/LogViewer/LogLine.tsx

```diff
@@ -159,8 +158,8 @@ export function LogLine({
   const elements = useMemo(
     () =>
       chunks.map(({ text, modifiers, highlight }, index) => (
-        <Typography
-          component="span"
+        // eslint-disable-next-line react/forbid-elements
```

Comment:
This is a slightly unrelated fix, that the styles are really all wrong when you wrap it in a mui element. So went back to how it was before.